### ISSUE TITLE
TimeManager.RoundTripTime is never updated

### ISSUE THREAD

**Issue by KeithSwanger** [FirstGearGames/FishNet#806]
**General**
Unity version: 2022.3.51
Fish-Networking version: 4.5.4

**Description**
`TimeManager.cs`

RoundTripTime is never updated due to a change made in 4.5.3.

 `_elapsedTickTime` is modified after the condition that checks to call `TrySendPing()`

Therefore, `_elapsedTickTime < timePerSimulation` is always false, and `TrySendPing()` is never called.

**Replication**
Debug.Log TimeManager.RoundTripTime. It will always be 0.

**Comment by FirstGearGames** [comment]
Will most likely release this as a hotfix without an incremental bump. 

Change this...
```cs
                    if (isClient && (_elapsedTickTime < timePerSimulation))
                        TrySendPing(LocalTick + 1);
```
to this...
```cs
                    if (canSendPing)
                    {
                        canSendPing = false;
                        TrySendPing(LocalTick + 1);
                    }
```
Add boolean just above the loop.
```cs
            bool canSendPing = (isClient && frameTicked);
            do
            {
```

**Comment by FirstGearGames** [comment]
Resolution is in TimeManager to replace this line...
```cs
                    if (isClient && (_elapsedTickTime < timePerSimulation))
```
with this...
```cs
                    bool lastTick = (_elapsedTickTime <= timePerSimulation);
                    if (isClient && lastTick)
```                    

Resolved in 4.5.4hf0
